README.md

```diff
@@ -170,6 +255,380 @@ then configure key protect service with authenticator. then run operations on se
             Response<GetImportToken> getResponse = testService.getImportToken(getImportTokenOptionsModel).execute();
             GetImportToken getResponseObj = getResponse.getResult();
 ```
+### List Registrations
+
+```
+            // Construct an instance of the GetRegistrationsOptions model
+            GetRegistrationsOptions getRegistrationsOptionsModel = new GetRegistrationsOptions.Builder()
+                     //.id(keyId)          // uncomment to list registrations for a key in the instance
+                    .bluemixInstance(bluemixInstance)
+                    .limit(Long.valueOf("50"))
+                    .totalCount(true)
+                    .build();
+
+            // Invoke operation with valid options model
+            Response<RegistrationWithTotalCount> response = testService.getRegistrations(getRegistrationsOptionsModel).execute();
+            RegistrationWithTotalCount responseObj = response.getResult();
+```
+
+### Set(create or update)/Retrieve key policies
+
+###### Set automatic rotation policy for a key
+```
+            // Construct an instance of the CollectionMetadata model
+            CollectionMetadata collectionMetadataModel = new CollectionMetadata.Builder()
+                    .collectionType("application/vnd.ibm.kms.policy+json")
+                    .collectionTotal(Long.valueOf("1"))
+                    .build();
+
+            // Construct an instance of the KeyPolicyRotationRotation model
```

Comment:
Add an additional comment or text here to say that we are building a policy that will cause keys to be automatically rotated every 2 months.